stt-worker/acquire.py

```
@@ -1,47 +1,51 @@
+    except ValueError:
+        return False
 
 
-def _download_media(url: str, out_dir: str) -> str:
-    """CDN 직다운. 확장자는 Content-Type(→URL경로)으로 결정해 영상/이미지 모두 처리."""
+def _download(url: str, out_dir: str) -> str:
+    """CDN 직다운. 확장자는 Content-Type(→URL경로)으로 결정해 영상/이미지 모두 처리.
+    만료 URL(oe 지남 또는 403/410)은 CdnExpiredError 로 구분해 던진다."""
+    if _is_expired(url):
+        raise CdnExpiredError("media_url 만료(oe 시각 지남)")
     req = urllib.request.Request(url, headers={"User-Agent": "Mozilla/5.0"})
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)